ci-operator/step-registry/aws/deprovision/users-and-policies/aws-deprovision-users-and-policies-ref.yaml

```diff
@@ -1,11 +1,11 @@
 ref:
-  as: aws-deprovision-cco-manual-users-static
```

Comment:
Seem like ci-operator/step-registry/quay-tests/aws-ipi-disconnected-private-deprovision/quay-tests-aws-ipi-disconnected-private-deprovision-chain.yaml also reference the step. cc @LiZhang19817